apps/examples/camera/camera_main.c

```diff
@@ -308,16 +708,30 @@ int camera_main(int argc, char *argv[])
 		goto errout_with_video_init;
 	}
 
+	if (cam_option) {
+		get_camera_options(v_fd, 1);
```

Comment:
you can put both get_camera_options(v_fd, 1); in the erroout_with_close?

Reply:
Adding "get_camera_options" function at the exit may have complication, if some depending resource has been released already.